<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageManage</class>
 <widget class="QWidget" name="MessageManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>群聊消息管理界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
background-color:#030408;
color:#db9a45;
font-size: 20px;
font:10pt &quot;Microsoft YaHei UI&quot;;
border-radius:8px;
font-weight:700;
height:30px;
width: 100%;
}
QPushButton:disabled {
background-color: #787a7e;
}
QPushButton:hover {
background-color: #383a3e;
}
QLineEdit{
border: 1px solid rgb(41, 57, 85);
border-radius: 3px;
border-color:#dcdee0;
height:36px;
background: white;
padding-left: 12px;
padding-right: 12px;
border-color:#e4e5eb;
}
QLineEdit:hover {
border-color: #db9a45;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="ContentInput">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="FindUserButton">
       <property name="text">
        <string>查询用户名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="FindContentButton">
       <property name="text">
        <string>查询消息内容</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DeleteButton">
       <property name="text">
        <string>撤回选中消息</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="ChatName">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="ChatID">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTableView" name="MessageTable"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ContentInput</tabstop>
  <tabstop>FindUserButton</tabstop>
  <tabstop>FindContentButton</tabstop>
  <tabstop>DeleteButton</tabstop>
  <tabstop>MessageTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
